[2021-10-05 11:59:29,647] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Accurate 2021-10-05T11:59:26.012521+00:00 [queued]>
[2021-10-05 11:59:29,675] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Accurate 2021-10-05T11:59:26.012521+00:00 [queued]>
[2021-10-05 11:59:29,677] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-05 11:59:29,678] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-05 11:59:29,679] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-05 11:59:29,693] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): Accurate> on 2021-10-05T11:59:26.012521+00:00
[2021-10-05 11:59:29,698] {standard_task_runner.py:52} INFO - Started process 74220 to run task
[2021-10-05 11:59:29,703] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Accurate', '2021-10-05T11:59:26.012521+00:00', '--job-id', '132', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpd3pvvjp7', '--error-file', '/tmp/tmp0u7_a432']
[2021-10-05 11:59:29,706] {standard_task_runner.py:77} INFO - Job 132: Subtask Accurate
[2021-10-05 11:59:29,764] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Accurate 2021-10-05T11:59:26.012521+00:00 [running]> on host d6b4131456ad
[2021-10-05 11:59:29,866] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Accurate
AIRFLOW_CTX_EXECUTION_DATE=2021-10-05T11:59:26.012521+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-05T11:59:26.012521+00:00
[2021-10-05 11:59:29,868] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-05 11:59:29,871] {subprocess.py:63} INFO - Running command: ['bash', '-c', "printf '============\n\nAccurate\n\n=============' "]
[2021-10-05 11:59:29,885] {subprocess.py:74} INFO - Output:
[2021-10-05 11:59:29,888] {subprocess.py:78} INFO - ============
[2021-10-05 11:59:29,889] {subprocess.py:78} INFO - 
[2021-10-05 11:59:29,891] {subprocess.py:78} INFO - Accurate
[2021-10-05 11:59:29,893] {subprocess.py:78} INFO - 
[2021-10-05 11:59:29,894] {subprocess.py:78} INFO - =============
[2021-10-05 11:59:29,895] {subprocess.py:82} INFO - Command exited with return code 0
[2021-10-05 11:59:29,953] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Accurate, execution_date=20211005T115926, start_date=20211005T115929, end_date=20211005T115929
[2021-10-05 11:59:30,002] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-05 11:59:30,043] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
